README.md

```diff
@@ -346,73 +255,64 @@ The following platforms are currently supported:
-virtual machine definition), there are a few switches which can make your life
-easier.
 
-  * `--debug=<level>`: Sets the verbosity of Krun.  Valid debug levels are:
-     `DEBUG`, `INFO`, `WARN`, `DEBUG`, `CRITICAL` and `ERROR`. The default is
-     `WARN`. For real benchmarks you should use the default.
+## Testing your configurations
 
-  * `--quick`: There are several places where Krun would normally wait using
-    sleeps or a polling loop. These are essential for real benchmarking, but
-    annoying for development. Use `--quick` to skip these delays.
+Before doing a full run of an experiment, you should perform a quick(ish) test
+of your configuration. This can be achieved with:
 
-  * `--no-user-change`: Without this flag, For each process execution, Krun
-    will use a fresh user account called 'krun'. This involves deleting any
-    existing user account (with `userdel -r`) and creating a new user account
-    (with `useradd -m`).  This switch disables the use of a fresh user account,
-    meaning that `userdel` and `useradd` are not invoked, nor does Krun switch
-    user; the user Krun was invoked with is used for benchmarking.
+```sh
+$ /path/to/krun/krun.py --dry-run --quick --debug=INFO config.krun
+```
 
-  * `--dry-run`: Fakes actual benchmark processes, making them finish
-    instantaneously.
+\laurie{i think this should have `--no-reboots` on too?}
 
-  * `--no-tickless-check`: Do not crash out if the Linux kernel is not
-    tickless.
-
-  * `--no-pstate-check`: Do not crash out if Intel P-states are not disabled.
+See the "Development and Debug Switches" section for a description of these
+switches.
 
-## Benchmarking for Reliable Results
 
-You should not collect results intended for publication with development switches
-turned on.
+## Production benchmarking
 
-You should also benchmark in "reboot mode". The recommended way to do
-this is via `cron(8)`. We supply a script `start_krun_from_cron` to make this
-easier.
+Achieving the highest possible benchmarking quality requires more care. First,
+none of Krun's debug or development switches must be used. Second, Krun needs to
+run in "reboot mode" where each process execution will be run after the machine
+has (automatically) rebooted. The recommended way to do this is via `cron(8)`
+and the `start_krun_from_cron` script supplied with Krun.
 
-As a regular user (who has access to your experiment and the Krun code), run
-`crontab -e` a line similar to the following:
+As a regular user (who has access to the experiment and the Krun code), run
+`crontab -e` and add a line similar to the following:
 
 ```
 @reboot /path/to/krun/scripts/start_krun_from_cron /path/to/your/config.krun
 ```
 
-The path to the config file must be an absolute path.
-
-Any arguments supplied after the config file path are passed to Krun unchanged.
+Note that the path to the config file must be an absolute path, otherwise `cron`
```

Comment:
The absolute path constraint isn't really to do with cron. It's a Krun constraint.